num256/Cargo.toml

```diff
@@ -9,4 +9,5 @@ num = "0.1.41"
 serde = "1.0.24"
 serde_derive = "1.0.24"
 serde_json = "1.0.8"
-lazy_static = "1.0.0"
\ No newline at end of file
+lazy_static = "1.0.0"
+ethereum-types = {git="https://github.com/paritytech/primitives.git"}
```

Comment:
We should probably be using a release of this, they have a fairly fresh `v0.3.2` tag (3 weeks old). What do you think?

Reply:
Our changes were merged in 7 days ago, so unfortunately not released yet

Reply:
Fair enough, approving